src/pretix/presale/templates/pretixpresale/event/checkout_payment.html

```diff
@@ -13,43 +13,49 @@
         <div class="panel-group" id="payment_accordion">
             {% for p in providers %}
                 <div class="panel panel-default" data-total="{{ p.total|floatformat:2 }}">
-                    <label class="accordion-radio">
+                    <div class="accordion-radio">
```

Comment:
To me, it looks like the text is not vertically centered with the radio button (screenshot taken in firefox):

![20210929_170742](https://user-images.githubusercontent.com/64280/135299545-20c06823-9d22-4a8a-a38b-7f50550a1da3.png)

Also, these panel heads are now much higher than other ones (which is fine for me, just something to be aware of)

![20210929_171030](https://user-images.githubusercontent.com/64280/135299649-ba4efb0c-c62f-452d-be1f-614497a1355e.png)

Reply:
Good catch! On staging this is currently not as bad, but not perfect either – panel heads are 41px instead of 38px high and the centering of the radio buttons is a bit off as well. I will fix this.

Reply:
Regarding your comment clicking the whole header instead of just the label/text: `<div>` is not allowed inside a `<label>`, so I had to move it. If it is okay, I will add a JavaScript helper for those label-headers. I am not a big fan of using JavaScript for this, but for being A11y compliant we need to be HTML compliant.

Reply:
Layout is fixed. JavaScript to trigger change-event on radio when clicking .accordion-radio is added.

Reply:
amazing :)